bot/src/auctions.ts

```diff
@@ -24,7 +26,15 @@ export const getNewAuctionsFromActiveAuctions = function (activeActions: Auction
 };
 
 export const getAllAuctions = async function (network: string): Promise<AuctionInitialInfo[]> {
-    const auctions = await fetchAllInitialAuctions(network);
+    let collateralWhitelist: undefined | string[];
+    if (WHITELISTED_COLLATERALS) {
+        collateralWhitelist = parseCollateralWhitelist(WHITELISTED_COLLATERALS);
+        console.info(`auctions: whitelist is enabled, only fetching auctions of type "${WHITELISTED_COLLATERALS}"`);
+    }
+
+    // if collateralWhitelist is undefined all auctions will be fetched
+    const auctions = await fetchAllInitialAuctions(network, collateralWhitelist);
```

Comment:
And then here you will execute `getWhitelistedCollaterals` in order to get a valid list or undefined
```suggestion
    const auctions = await fetchAllInitialAuctions(network, getWhitelistedCollaterals());
```